test/jdk/java/nio/file/Files/probeContentType/Basic.java

```diff
@@ -188,6 +190,15 @@ public static void main(String[] args) throws IOException {
         };
         failures += checkContentTypes(exTypes);
 
+        // Verify type is found when the extension is in a fragment component
+        Path pathWithFragement = Path.of("SomePathWith#aFragement.png");
+        String contentType = Files.probeContentType(pathWithFragement);
+        if (contentType == null || !contentType.equals("image/png")) {
+            System.out.printf("For %s expected \"png\" but got %s%n",
```

Comment:
Should this instead use `System.err`, since the rest of the test uses that for printing these failures when incrementing the failure count.

Reply:
Yes, it should have used `System.err`; fixed in eab33e8000ea730c57c918dbf291af23bacbd059.